pkg/lib/cockpit-components-shutdown.jsx

```diff
@@ -109,7 +104,7 @@ export class ShutdownModal extends React.Component {
             return;
         }
 
-        const cmd = ["date", "--date=" + moment(date).format('YYYY-MM-DD') + " " + this.state.time, "+%s"];
+        const cmd = ["date", "--date=" + (new Intl.DateTimeFormat().format(this.state.dateObject)) + " " + this.state.time, "+%s"];
```

Comment:
That DateTimeFormat() will output US format, as no locale is given. I confirmed that `date --date` does accept that in non-English locales as well.